.github/workflows/e2e.yml

```diff
@@ -0,0 +1,83 @@
+# Copyright © 2023 Cask Data, Inc.
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+# Note: Any changes to this workflow would be used only after merging into develop
+name: Build e2e tests
+
+on:
+  workflow_run:
+    workflows:
+      - Trigger build
+    types:
+      - completed
+
+jobs:
+  build:
+    runs-on: k8s-runner-e2e
+
+    if: ${{ github.event.workflow_run.conclusion != 'skipped' }}
+
+    steps:
+      # Pinned 1.0.0 version
+      - uses: haya14busa/action-workflow_run-status@967ed83efa565c257675ed70cfe5231f062ddd94
```

Comment:
use `marocchino/action-workflow_run-status@54b6e87d6cb552fc5f36dbe9a722a6048725917a` to avoid node 12 warning which is deprecated.

Reply:
Please update this workflow as well: https://github.com/data-integrations/amazon-s3-plugins/blob/56a6186f4ffac32480fe1129e6bab7d9e176b4e1/.github/workflows/build.yml#L32

Reply:
change done, as suggested.